addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -2864,6 +2865,7 @@ QUnit.module("Views", (hooks) => {
     QUnit.test('graph view with attribute disable_linking="1"', async function (assert) {
         assert.expect(4);
 
+        serviceRegistry.add("localization", makeFakeLocalizationService());
```

Comment:
I don't understand why you need this in this commit

Reply:
we use _t to default the title. In this test, we mock the action service, so we don't have localization automatically